docs/reference/announcements.md

```diff
@@ -91,7 +91,17 @@ Version changes are made to supported environments:
 
 To learn more about supported environments, see [supported environments](/reference/supported-environments.md).
 
-### Breaking changes in the Connector SDK
+### Connectors
```

Comment:
Given the terminology of element template here, is this applicable only to Desktop Modeler? If so, we should likely also add a note in that documentation.

Reply:
Actually it's applicable to both Desktop and Web Modelers - Web Modeler users will no longer see the "plain start event" connector option for Kafka, RMQ, SQS, and SNS when appending a template to an event.

Reply:
Given this is not exclusive to Desktop Modeler then, should element template be called Connector template in these instances?

Reply:
@christinaausley AFAIK element template is the way to go.